page-factory-core/src/main/java/ru/sbtqa/tag/pagefactory/data/DataFactory.java

```diff
@@ -65,10 +66,10 @@ public static TestDataProvider getDataProvider() throws DataException {
                     );
                     break;
                 case "mongo":
-                    testDataProvider = initProvider(PROVIDERS.MONGO_DATA_PROVIDER,
-                            new MongoClient(new MongoClientURI(Props.get("data.uri"))).getDB("data.db"),
-                            initialCollection
-                    );
+                    MongoClient mongoClient = new MongoClient(new MongoClientURI(Props.get("data.uri")));
+                    DB db = mongoClient.getDB(Props.get("data.db"));
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this use of "getDB"; it is deprecated. ([squid:CallToDeprecatedMethod](https://sonarcloud.io/organizations/sbtqasonarkey/rules#rule_key=squid%3ACallToDeprecatedMethod))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=ru.sbtqa.tag%3Apage-factory-2&pullRequest=92&issues=AWYu57qovOWp7Yd6SF6d&open=AWYu57qovOWp7Yd6SF6d)

[ISSUE_KEY]: AWYu57qovOWp7Yd6SF6d